gdk/PixbufLoader.custom

```diff
@@ -141,3 +147,30 @@
 		{
 			return new PixbufLoader (System.Reflection.Assembly.GetCallingAssembly (), resource);
 		}
+
+        class ByteArrayPool
+        {
+            WeakReference<System.Collections.Concurrent.ConcurrentQueue<byte[]>> queuedBuffers =
```

Comment:
This is interesting. Just for my education:

1. Why did you make it a weak reference? You don't want to hold the buffers in memory?
2. Why a queue? Why not just a single buffer?